src/access/universal.ts

```diff
@@ -166,6 +170,44 @@ export async function setAgentAccess(
   return null;
 }
 
+/**
+ * Get an overview of what access are defined for all Agents with respect to a given
```

Comment:
```suggestion
 * Get an overview of what access is defined for all Agents with respect to a given
```

Access is uncountable too I think :)